bluemira/equilibria/flux_surfaces.py

```diff
@@ -123,6 +123,51 @@ def connection_length(self, eq):
         """
         return np.sum(self._dl(eq))
 
+    def calculate_poloidal_angle(self, eq, x_strike, z_strike, gamma):
+        """
+        From glancing angle to poloidal angle.
+
+        Parameters
+        ----------
+        eq: Equilibrium
+            Equilibrium from which to take magnetic fields
+            at the strike pointcalculate the flux expansion
+        x_strike: float
+            x coordinate of strike point
+        z_strike: float
+            z coordinate of strike point
+        gamma: float
+            Glancing angle at the strike point [deg]
+
+        Returns
+        -------
+        theta: float
+            Poloidal angle at the strike point [deg]
+        """
+        # Technological limit
+        if gamma < 3:
+            raise ValueError(
+                "Glancing angle below the technological limit of 3 degrees."
+            )
```

Comment:
I am reluctant on raising a true error that halts the code on this angle. The reason is that that technological developpement can actually allow lower angle in the future and we want to remain open to that. The user can also purpusefully use unrealistic angle for scoping studies and see if it is worth finding novel ways on improving on the the angle.

A Bluemira error that does not halts the code but notify the user is more appropritate.